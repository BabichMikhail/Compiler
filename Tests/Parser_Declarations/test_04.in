const 
c : integer = 10;