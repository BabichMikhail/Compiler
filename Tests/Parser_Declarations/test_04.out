Const	c	integer
10

